src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/ComponentDemo.razor

```diff
@@ -33,14 +33,17 @@
             <ComponentParametersTable Name="BitInputBase" Parameters="_inputBaseParameters" />
         }
 
-        <ComponentParametersTable Name="BitComponentBase" Parameters="_componentBaseParameters" />
+        @if (ComponentBaseParameters is not null)
```

Comment:
good point 👌
but this change is not related to this issue, please roll it back.